bin/crontabScripts/clearIndex.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#:Title: clearIndex.sh
+#:Author: Alan Coleman
+
+#clears content stored in "my_index"
+curl -XDELETE 'localhost:9200/my_index'
```

Comment:
Do you need to check if elasticsearch is up and running? 
Where do they define the logs for the corn jobs? If not available out of box we need to add logging.

Reply:
elasticsearch needs to be running for the index to be cleared, the user receives notification about whether the index has been cleared or if elasticsearch was not running

Reply:
In that case, it should be ok.